optaplanner-wb-screens/optaplanner-wb-guided-rule-editor/optaplanner-wb-guided-rule-editor-backend/src/test/java/org/optaplanner/workbench/screens/guidedrule/backend/server/plugin/HardConstraintMatchPersistenceExtensionTest.java

```diff
@@ -70,4 +70,14 @@ public void unmarshalActionBendableHardConstraintMatch() throws RuleModelDRLPers
     public void unmarshalUnrecognizedString() throws RuleModelDRLPersistenceException {
         extension.unmarshal("unrecognizedString");
     }
+
+    @Test(expected = RuleModelDRLPersistenceException.class)
```

Comment:
Although this approach for exception checking might be fine here, you cannot check e.g. the error message. I find AssertJ especially for this purpose more interesting:
http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion